src/app/components/media/MediaComponent.js

```diff
@@ -143,94 +212,175 @@ class MediaComponent extends Component {
     media.quote = media.media.quote;
     media.embed_path = media.media.embed_path;
 
+    const {
+      fragment,
+      duration,
+      playerRect,
+      playing,
+      progress,
+      start, end, gaps,
+      scrubTo,
+      seekTo,
+      showVideoAnno,
+      time,
+    } = this.state;
+
+    const { currentUser } = this.getContext();
+
+    // console.group("MediaComponent.js");
+    // console.log(this.state);
+    // console.groupEnd();
+
     return (
-      <React.Fragment>
+      <LoadScript
+        googleMapsApiKey={window.config.googleMapsApiKey}
+        libraries={GOOGLE_MAPS_LIBRARIES}
```

Comment:
If this constant is not used anywhere else, can we just put the straight value in here?

Reply:
nah, react complains that forcers some re-render as `["places"]` identity changes on each render